build.gradle

```diff
@@ -69,6 +69,7 @@ subprojects { subproject ->
 	apply from:   "${rootDir}/src/checkstyle/fixModifiers.gradle"
 	apply from:   "${rootDir}/src/checkstyle/fixThis.gradle"
 	apply from:   "${rootDir}/src/checkstyle/fixRightCurly.gradle"
+	apply from:   "${rootDir}/src/checkstyle/fixWhiteAround.gradle"
```

Comment:
There is no such a script in the PR.
From other side it is so big, that I would like to see only the script and let you merge the final result on your own.